tests/server/ShellUTest.cxxtest

```diff
@@ -192,7 +192,8 @@ public:
 	ShellUTest()
 	{
 		logger().set_level(Logger::DEBUG);
-		//logger().set_print_to_stdout_flag(true);
+		logger().set_print_to_stdout_flag(true);
```

Comment:
you probably don't want to do that -- it generates megabytes of highly repetitive gunk.